src/redux/actions/experiments/createExperiment.js

```diff
@@ -21,7 +21,7 @@ const createExperiment = (
     id: hash.MD5(createdAt),
     name: newExperimentName,
     projectUuid,
-    createdAt,
+    createdDate: createdAt,
```

Comment:
I would rename the variable `createdAt` so that here we could use the short form for attribute initialisation.

Reply:
This was a problem that I made. The other objects `projects`, `samples`, uses `createdDate` for the name of this field. Only `experiments` use `cratedAt`, which should have been `createdDate`. For the sake of consistency, I think it's better to use `createdDate` while we haven't used data-management fully.

Reply:
Sorry, I meant to change above 
```
const createdDate = moment().toISOString();
```

Reply:
It's also used to create the `id: hash.MD5(createdAt)`, that's why it's refactored out,

Reply:
I still haven't been able to make clear my TINY MINOR point :-D
```
  const createdDate = moment().toISOString();

  const newExperiment = {
    ...experimentTemplate,
    id: hash.MD5(createdDate),
    name: newExperimentName,
    projectUuid,
    createdDate,
  };
```

Reply:
oh ok hahaha